.github/workflows/demo.yml

```
@@ -0,0 +1,135 @@
+      - name: Install Atmos
+        uses: jaxxstorm/action-install-gh-release@v1.11.0
+        with:
+          repo: cloudposse/atmos
+          tag: ${{ needs.prepare.outputs.version }}
+          chmod: 0755
+          extension-matching: disable
+          rename-to: atmos
+
+      - name: Prepare variables
+        id: vars
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)